src/sentry/static/sentry/less/includes/select2.less

```diff
@@ -11,7 +11,6 @@
   padding: 4px 10px 3px;
   top: -1px;
   box-shadow: 0 1px 1px rgba(0, 0, 0, .05);
-  max-width: 180px;
```

Comment:
@ckj fyi this changes some pages -- full width select2 isn't ideal, but we can fix them by just setting a width (vs limiting what select2 can do in general). I didn't think it was bad enough to go change them.
![screenshot 2016-04-15 11 02 16](https://cloud.githubusercontent.com/assets/23610/14570516/889cc43c-02f9-11e6-9d1e-600877db079f.png)